democracy_club/templates/hermes/post_detail.html

```diff
@@ -6,11 +6,16 @@
 {% block title %}{{ post.subject }} | {{ site_title }}{% endblock title %}
 
 {% block og_description_content %}{{ post.summary }}{% endblock og_description_content %}
-{% block og_image_content %}{% thumbnail post.hero "690" as im %}{{ im.url }}{% endthumbnail %}
+{% block og_image_content %}{% thumbnail post.hero "690" as im %}{{ CANONICAL_URL }}{{ im.url }}{% endthumbnail %}
 {% endblock og_image_content %}
 {% block og_title_content %}{{ post.subject }}{% endblock og_title_content %}
 {% block og_type_content %}article{% endblock og_type_content %}
 
+ <!-- Twitter -->
+{% block og_image_twitter_content %}{% thumbnail post.hero "690" as im %}{{ CANONICAL_URL }}{{ im.request }}{% endthumbnail %}{% endblock og_image_twitter_content %} 
```

Comment:
Remove the `{{ CANONICAL_URL }}` from this and I think in staging/production it will build the full url because it is using S3